src/client/game.cpp

```diff
@@ -1139,8 +1148,28 @@ bool Game::startup(bool *kill,
 	if (!createClient(start_data))
 		return false;
 
-	if(start_data.isRecording())
+	// create ZMQ objects
+	std::string address = std::string("tcp://127.0.0.1:") + start_data.dumb_client_port;
```

Comment:
What if we want to connect to a remote machine? Possible solution: host a router on the host PC.
Also, this removes the need for `dumb-port`. If this change goes through, the two arguments have to be renamed and merged.

Reply:
Yes, good point. I mean we can still have a configurable address, maybe just rename `dumb-port` to `dumb-address`?